code/src/containers/forms/product-form/components/additional-info/AdditionalInfo.tsx

```diff
@@ -0,0 +1,110 @@
+import { Controller, ControllerRenderProps } from "react-hook-form";
+
+import { productCategories } from "@/containers/forms/product-form/ProductForm.constants";
+import {
+  ProductFormAdditionalInfoSectionProps,
+  ProductFormValues
+} from "@/containers/forms/product-form/ProductForm.types";
+
+import AppBox from "@/components/app-box/AppBox";
+import AppCheckbox from "@/components/app-checkbox/AppCheckbox";
+import AppFormHelperText from "@/components/app-form-helper-text/AppFormHelperText";
+import AppInput from "@/components/app-input/AppInput";
+import AppMenuItem from "@/components/app-menu-item/AppMenuItem";
+import AppSelect from "@/components/app-select/AppSelect";
+import AppTypography from "@/components/app-typography/AppTypography";
+
+const AdditionalInfo = ({
+  register,
+  errors,
+  control
+}: ProductFormAdditionalInfoSectionProps) => {
+  const categoryErrorElement = errors.category && (
+    <AppFormHelperText className="product-form__category-select-helper" error>
+      {errors.category.message}
+    </AppFormHelperText>
+  );
+
+  const selectControllerRenderFunction = ({
+    field: handlers
+  }: {
```

Comment:
Move to local type to improve readability

Reply:
I made that part more readable